docs/src/app/components/pages/components/progress.jsx

```diff
@@ -16,7 +16,7 @@ const ProgressPage = React.createClass({
   componentDidMount() {
     let self = this;
```

Comment:
:scream: :scream: I didn't notice this ugly monster O.O

I'll remove all of them when I see them.